source/team.html.erb

```diff
@@ -144,7 +144,7 @@ responsive: true
     The Ember Learning Team
   </h2>
   <p class="lead text-center">
-    The Ember Learning Team is responsible for keeping the guides and API documentation accurate and up-to-date. They also manage the community outreach coordination and other initiatives that support learning Ember.
+    The Ember Learning Team empowers Ember users to learn, build and teach. They are responsible for keeping the guides and API documentation accurate and up-to-date. They also manage the community outreach coordination and other initiatives that support learning Ember.
```

Comment:
If it's not too much trouble, would you mind changing the text a little bit? I tried to streamline it so it was a little more efficient:

The mission of the Ember Learning Team is to empower Ember users to learn, build and teach. This team is responsible for keeping the guides and API documentation updated, and manages the initiatives that support learning Ember.